rtengine/ipwavelet.cc

```diff
@@ -2032,7 +2032,8 @@ void ImProcFunctions::WaveletcontAllL(LabImage * labco, float ** varhue, float *
 
                         if(cp.lip3 && cp.lipp) {
                             // comparaison betwwen pixel and neighbours
-                            float kneigh, somm;
```

Comment:
Again, making taking one of the initialization branches seems preferable. Rewriting this using the ternary `?:` operation and giving those variables the type `const auto` might be advisable as well.